docs/path-for-tracked-resource-types.md

```diff
@@ -0,0 +1,33 @@
+# PathForTrackedResourceTypes
```

Comment:
>PathForTrackedResourceTypes [](http://example.com/codeflow?start=2&length=27)

It is best to introduce one change per PR so the changelog and commit history is clean

Reply:
Sure, will not do it from the next PR